compiler/AST/checkAST.cpp

```diff
@@ -214,8 +216,6 @@ void checkPrimitives()
      case PRIM_ARRAY_SHIFT_BASE_POINTER:
      case PRIM_ARRAY_SET:
      case PRIM_ARRAY_SET_FIRST:
-     case PRIM_ERROR:
-     case PRIM_WARNING:
```

Comment:
Before I changed it, checkPrimitives() just ignored PRIM_WARNING. The switch case falls through to a 'break' on master. PRIM_WARNING can (and did, in testing) exist past resolution.